subsys/bluetooth/host/gatt.c

```diff
@@ -2110,16 +2132,42 @@ static bool gatt_cf_notify_multi(struct bt_conn *conn)
 	return CF_NOTIFY_MULTI(cfg);
 }
 
+static int gatt_notify_flush(struct bt_conn *conn)
+{
+	int err = 0;
+	struct net_buf **buf = &nfy_mult[bt_conn_index(conn)];
+
+	if (*buf) {
+		err = gatt_notify_mult_send(conn, buf);
+	}
```

Comment:
Minor, but this may be slightly better

```suggestion
	struct net_buf *buf = nfy_mult[bt_conn_index(conn)];

	if (buf) {
		err = gatt_notify_mult_send(conn, &buf);
	}
```

Reply:
No. I think that's incorrect. The semantics are different. With the suggested change `gatt_notify_mult_send` will get a pointer to `buf` instead of a pointer to the entry in `nfy_mult`.

Reply:
@alwa-nordic changed the signature to `mult_send`, see the new revision. This particular block right here has to keep the double-ptr though.